src/tests/test_contributor.py

```diff
@@ -0,0 +1,104 @@
+import pdb
+import unittest
+import os
+import json
+# import flask_sqlalchemy
+from ..models.Contributor import Contributor
+from ..models.Campaign import Campaign
+from ..models.CampaignContributor import CampaignContributor
+from ..app import create_app, db
+
+class ContributorTest(unittest.TestCase):
+  # set up the tests with 2 contributors
+  # This runs before EVERY test. not once before all
+  def setUp(self):
+    self.app = create_app('testing')
+    self.client = self.app.test_client
+    self.contributor1 = {
+        "address": "XyZpekdA",
+        "email": "test@email.com"
+    }
+    self.contributor2 = {
+        "address": "AaBbCc9"
+    }
+
+    self.campaign1 = {
+        "name": "Test Campaign",
+        "description": "test campaign description",
+        "image": "test.jpg",
+        "upvote": 2,
+        "manager": "XYZ",
+        "address": "X73K",
+        "min_contribution": 5.0
+    }
+
+    with self.app.app_context():
+      # create all db objects
+      db.drop_all()
+      db.create_all()
+      # delete test campaigns by name
+      existing_contributor1 = Contributor.get_contributor_by_address(self.contributor1["address"])
+      if existing_contributor1:
+        existing_contributor1.delete()
+      existing_contributor2 = Contributor.get_contributor_by_address(self.contributor2["address"])
```

Comment:
I've found that `db.drop_all()` takes care of the `existing_object` check. I'm fairly sure you can delete lines 40-45